rust/src/lib.rs

```diff
@@ -260,8 +258,8 @@ pub extern "C" fn zcashlc_init_accounts_table(
                         CString::new(encoded).unwrap().into_raw()
                     })
                     .collect();
+                v.shrink_to_fit();
```

Comment:
This doesn't work. Per the [documentation for `Vec::shrink_to_fit`](https://doc.rust-lang.org/stable/std/vec/struct.Vec.html#method.shrink_to_fit):

> It will drop down as close as possible to the length but the allocator may still inform the vector that there is space for a few more elements.